docs/sources/k6/next/reference/versioning-and-stability-guarantees.md

```diff
@@ -0,0 +1,285 @@
+* **The actively maintained** version is the latest released major version of k6.
+* **The supported** version is the latest released version of k6 within the previous major version.
+* **The deprecated** versions are the set of previously released k6 versions.
+
+As such, during its development and support workflow, the k6 open source project actively develops the next version of k6, maintains the latest major release, supports the previous major version, and doesn't provide any support for releases older than the supported one.
+
+Any past version of k6 that’s neither being developed, maintained, or supported is considered **deprecated** and will not support further updates or development. Users are free to use them at their own risk.
+
+### Guidelines
+
+1. _Features are not backported_ from one major version to any previous ones.
+2. Only changes on the latest release of each support category are provided: _at any point in time, there are single officially under-development and actively maintained versions_. We do not support multiple versions of each support path. For example, suppose a bug is found in v1.1.0, and the latest v1.Y.Z published release is v1.3.2. In that case, if and only if the latest release is affected, and the bug or security issue is deemed important enough, only then does the development team define as a priority to publish a fix that will be released in v1.3.3. Users cannot expect a v1.1.1 fix release.
+3. Other Grafana products, services, and teams building upon k6 are expected to develop their versioning integration built on our support policies.
+
+#### Bugs & Security issues
+
+Bugs and security issues come in all shapes and sizes and vary widely in impact. Some are critical, compromising user safety or impacting software functionality, while others are minor annoyances that do not require immediate action.
+
+The k6 open-source core team, with input from affected stakeholders, is responsible for assessing the severity of these issues.
+
+We classify bug severity based on their impact on users or product functionality. High-severity bugs prevent a subset of users from using the software properly or at all. Depending on the severity, the team will either release an immediate patch or, for less critical bugs, delay the fix to the next scheduled release.
+
+Security issues are classified using the CVSS (Common Vulnerability Scoring System) score when available. In its absence, we evaluate the threat in collaboration with the security team and affected teams. This assessment considers the potential threat to customers, users, system health, and data integrity. Like bug fixes, the response will be either an immediate patch release for high-severity issues or a delayed fix for lower-severity issues.
+
+Following the practice of the Grafana project, *security-related releases follow the same process but* ***MUST*** *be prepared in secret. They* ***MUST NOT*** *include changes that are not related to the security fix.*
```

Comment:
Do we have a plan here? Are we planning to have a "private" fork?

Reply:
This is probably very internal and not really needed to be here.

```suggestion
```